aion/web/models/question.ex

```diff
@@ -36,6 +36,16 @@ defmodule Aion.Question do
     Repo.get(Question, question_id)
   end
 
+  @spec get_questions_by_room_id(integer) :: Question.t
+  def get_questions_by_room_id(room_id) do
+    Repo.all(
+      from q in Question,
+      join: rc in RoomCategory, on: rc.category_id == q.category_id,
+      join: r in Room, on: r.id == rc.room_id,
+      where: r.id == ^room_id
```

Comment:
Not sure if the query macros work well with trailing commas, but if they do, you can put it here

Reply:
I followed documentation from Ecto.Query. They used it this way so I guess it's fine :) If you have better conventio to do it just let me know 👍

Reply:
Well if they do it this way in Ecto's docs, let's stay with that for now :)